python/src/aac/plugins/validators/root_keys/_validate_root_keys.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from aac.lang.language_context import LanguageContext
+from aac.lang.active_context_lifecycle_manager import get_active_context
+from aac.lang.definitions.definition import Definition
+from aac.plugins.validators import ValidatorResult
+
+
+def validate_root_keys(definition_under_test: Definition, target_schema_definition: Definition, active_context: LanguageContext, *validation_args) -> ValidatorResult:
```

Comment:
I've noticed a few places where we alternate between using the `active_context` parameter and calling `get_active_context()`. I expect this is effectively the same in our tests because we're always resetting the general active context but I wonder if it makes sense for us to be passing `active_context` if we're going to default to getting the active context via the `get_active_context()` function?

Having both options seems error-prone.

Reply:
Good call out. I think the fix is to rename arguments from `active_context` to `language_context`, which should leave `get_active_context()` as the only reference to the Active Context. 

This change will need to be done across all of the validator plugins -- we should coordinate before I make that change in the PR, or we can make a small issue for it.

Reply:
I think creating a small dedicated issue for it would be the best approach, personally.